recipes/integron_finder/meta.yaml

```diff
@@ -9,6 +9,8 @@ build:
   noarch: python
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
 
 source:
   url: https://files.pythonhosted.org/packages/4a/39/816165ed051a41560c9167590bf974a83de34e5a43509e9501f79cdf4ba4/integron_finder-2.0.2.tar.gz 
```

Comment:
version does not match with the version in line 2. please use syntax like following if possible
```
https://files.pythonhosted.org/packages/4a/39/816165ed051a41560c9167590bf974a83de34e5a43509e9501f79cdf4ba4/{{ name }}-{{ version }}}.tar.gz
```